zproject/backends.py

```diff
@@ -2295,6 +2296,22 @@ def user_data(self, access_token: str, *args: Any, **kwargs: Any) -> dict[str, s
         raise AssertionError("Invalid configuration")
 
 
+@external_auth_method
+class AzureADB2CAuthBackend(SocialAuthMixin, AzureADB2COAuth2):
+    # Allows authentication via Azure B2C in tenants
+
+    sort_order = 40
+    name = "azuread-b2c-oauth2"
+    auth_backend_name = "AzureADB2C"
+    display_icon = staticfiles_storage.url("images/authentication_backends/azuread-icon.png")
+
+    available_for_cloud_plans = [
+        Realm.PLAN_TYPE_STANDARD,
+        Realm.PLAN_TYPE_STANDARD_FREE,
+        Realm.PLAN_TYPE_PLUS,
+    ]
```

Comment:
due to the tenant-specific nature of this backend, I don't think it'll make sense to enable for Zulip Cloud, so `available_for_cloud_plans`  isn't needed. can you  remove it?